### PR TITLE
Add openSUSE-release package to all openSUSE images

### PR DESCRIPTION
**Pull request by sometimesfood**
This pull request adds the `openSUSE-release` package to all openSUSE Docker images.

Without `openSUSE-release`, `/etc/os-release` is not available, which makes distribution detection quite hard for the official openSUSE images. So far all the distributions I checked (CentOS, Fedora, Debian, Ubuntu, Oracle Linux, Alpine) had `/etc/os-release` in their official Docker images, so it would be great if openSUSE could include `os-release` too.

### CODE REVIEW

**Comment by flavio** [comment]
LGTM, thanks for the contribution. I'll merge the request when I'm back to a place with better internet connection (I'm travelling right now). I'll take care of pushing the new images to Docker Hub

**Comment by sometimesfood** [comment]
Great to hear that, thanks!

**Comment by flavio** [comment]
I'm tired of taking care of these image rebuilds manually. I ported your changes on OBS where we are going to automatically build these images. 

See https://build.opensuse.org/project/show/Virtualization:containers:images and its subproject.

I'll take care of pushing the build results of OBS to the Docker Hub. I'll also update this repo to drop all these sources, from now on we will do everything on OBS.

**Comment by sometimesfood** [comment]
Wouldn't it be possible to trigger the OBS builds via a GitHub webhook or service hook?

While I can imagine that OBS makes your life as a maintainer a lot easier with regards to image updates, simply being able to contribute via GitHub is hard to beat in terms of simplicity.

**Comment by adrianschroeter** [comment]
Yes it is possible: http://openbuildservice.org/2013/11/22/Source-Update-Via_Token/

**Comment by sometimesfood** [comment]
OK, let me rephrase then: I think it would be great for users to be able to keep using GitHub when submitting contributions.

I am sure OBS is a great tool, but only a small number of potential contributors is going to be as familiar with it as they are with GitHub.

**Comment by dmacvicar** [comment]
May be we can keep the files here and use a git service in OBS to pull them?

**Comment by flavio** [comment]
Good idea, I'll take care of that.

**Comment by sometimesfood** [comment]
Very glad to hear that, thanks! :+1:

**Comment by flavio** [comment]
Fixed. Unfortunately it is not possible to have the git hook because this repository defines two different OBS projects. I already talked with @adrianschroeter about that :(

**Comment by dmacvicar** [comment]
That would be for a automatic hook, but we can always do osc service disabledrun or? It was not automatic until now either.

**Comment by flavio** [comment]
We just have to trigger a rebuild of the project, that will automatically fetch latest version of the code from this repository. So yes, the situation definitely improved.

**Comment by sometimesfood** [comment]
@flavio Could you also push the updated images to docker-library/official-images? Thanks!

**Comment by flavio** [comment]
@sometimesfood yes, yesterday I didn't have time to do that. I'll take care of that today

**Comment by flavio** [comment]
@sometimesfood tracked here https://github.com/docker-library/official-images/pull/853

**Comment by sometimesfood** [comment]
Great, thanks!
